playfab-docs/features/automation/cloudscript-af/quickstart.md

```diff
@@ -50,8 +55,6 @@ To begin using this feature you need to enable it.  Visit Automation -> CloudScr
 
 ### Using and Calling CloudScript using Azure Functions from your PlayFab Title
 
-In order to use the function you just created from the PlayFab SDK, you will need to download one of our BetaSDKs.  These are gated SDKs, and you will need to request access to them via [helloplayfab@microsoft.com](mailto:helloplayfab@microsoft.com).  Once you have access to the [Beta SDKs](https://github.com/PlayFab?utf8=%E2%9C%93&q=beta&type=&language=) you can download the one specifically for the environment you are making your game in.  
```

Comment:
Why did you remove this text?  They do need the beta SDK so we need to provide them some instructions.

Reply:
The choice is 1) we're not giving new titles access right now, so no need for instructions, or 2) provide them a way to request access, in which case it would be a ticket to the support team who escalate to the feature team. Those were the options I was talking about in Teams earlier. Since I didn't hear back, I decided to go with 1. If you prefer 2, I can do that.

Reply:
We are giving new titles access but there is a bar where we aren't giving everyone.  I am ok to remove this as I am also adding in the welcome email but generally prefer to keep this so that if someone who didn't get the welcome email (or didn't read it) is actually doing the work.